src/vos/vos_internal.h

```diff
@@ -224,14 +224,15 @@ struct vos_space_metrics {
 
 /* VOS Pool metrics for WAL */
 struct vos_wal_metrics {
-	struct d_tm_node_t      *vwm_wal_sz;		/* WAL size for single tx */
-	struct d_tm_node_t      *vwm_wal_qd;		/* WAL transaction queue depth */
-	struct d_tm_node_t      *vwm_wal_waiters;	/* Waiters for WAL reclaiming */
-	struct d_tm_node_t	*vwm_replay_size;	/* WAL replay size in bytes */
-	struct d_tm_node_t	*vwm_replay_time;	/* WAL replay time in us */
-	struct d_tm_node_t	*vwm_replay_count;	/* Total replay count */
-	struct d_tm_node_t	*vwm_replay_tx;		/* Total replayed TX count */
-	struct d_tm_node_t	*vwm_replay_ent;	/* Total replayed entry count */
+	struct d_tm_node_t *vwm_wal_sz;                          /* WAL size for single tx */
+	struct d_tm_node_t *vwm_wal_qd;                          /* WAL transaction queue depth */
+	struct d_tm_node_t *vwm_wal_waiters;                     /* Waiters for WAL reclaiming */
+	struct d_tm_node_t *vwm_wal_lat[D_TM_IO_LAT_BUCKETS_NR]; /* WAL commit latency */
```

Comment:
It's not necessary to introduce WAL latency metrics for each io size, single WAL latency should be sufficient for the latency measurement use case.

Reply:
- [ ] Use single WAL latency

Reply:
> * [x]  Use single WAL latency

Fixed with commit 9ed8e926e